src/VaultSharp/V1/SecretsEngines/OpenLDAP/Role.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Text.Json.Serialization;
+
+namespace VaultSharp.V1.SecretsEngines.OpenLDAP
+{
+    public class Role
+    {
+        /// <summary>
+        /// A templatized LDIF string used to create a user account.
+        /// </summary>
+        /// <remarks>
+        /// This may contain multiple LDIF entries. The creation_ldif can also be used to add the
+        /// user account to an existing group. All LDIF entries are performed in order. If Vault
+        /// encounters an error while executing the creation_ldif it will stop at the first error
+        /// and not execute any remaining LDIF entries. If an error occurs and rollback_ldif is
+        /// specified, the LDIF entries in rollback_ldif will be executed. See rollback_ldif for
+        /// more details. This field may optionally be provided as a base64 encoded string.
+        /// </remarks>
+        [JsonPropertyName("creation_ldif")]
+        public string CreationLdif { get; set; }
```

Comment:
Recommendation to capitalise all instances of Ldif with LDIF since it is an abbreviation.